doc/platforms.md

```diff
@@ -2,26 +2,26 @@
 
 Flame runs on top of Flutter, so its supported platforms depend on how mature/stable support for each platform is by Flutter.
 
-At the moment, mobile has full support, and we try to keep supporting web the best we can.
+At the moment, Flame supports both mobile and web.
```

Comment:
Maybe we should mention that it works on Desktop as well now that it is on stable as well, wdyt?

Reply:
Its hidden behind a flag, and despite it being there I personally have not tested it out. So is it wise to say that we support it, if we are not testing it ourself?

Reply:
Since Flame v1 is just plain Flutter, I would say that it is safe to say that it work, but since it is behind a flag, maybe you are correct. Maybe we could just quote that it should work on web, but we are focusing our support efforts on mobile and web for now?

But either way, this don't need to be done on this Pr, we can discuss desktop later

Reply:
Totally, and as we already support both mobile and web I think we can resolve this convo?